Directory.Build.props

```diff
@@ -1,11 +1,11 @@
 <Project>
     <PropertyGroup>
         <!--todo: add mono-->
-        <TargetFramework>net8.0</TargetFramework>
+        <TargetFramework>net10.0</TargetFramework>
```

Comment:
We need to introduce one more Directory.Build.props inside 'src' directory. There we can set net10. This file will be overwritten